makefiles/defaultmodules.inc.mk

```diff
@@ -1,3 +1,7 @@
 DEFAULT_MODULE += board cpu core core_msg sys
 
 DEFAULT_MODULE += auto_init
+
+ifneq (,$(filter ssp,$(FEATURES_PROVIDED)))
+  DEFAULT_MODULE += ssp
+endif
```

Comment:
Why not use `FEATURES_OPTIONAL += ssp`?

Reply:
`DEFAULT_MODULE`s can be disabled via `DISABLE_MODULE`, I think that's not possible with features.

Reply:
You can, using `FEATURES_BLACKLIST += <FEATURE_TO_BLACKLIST>` ;-)